jobs/e2e-test-cypress.groovy

```diff
@@ -92,28 +93,42 @@ def runLamdaTest() {
       get qvYpo_KnpCiLBN69fYpEYA --format json | jq -r .password\
       """, returnStdout:true).trim();
    
-   // TODO(ruslan): implement TEST_TYPE param to use decorator or flag 
+   // TODO(ruslan): Implement TEST_TYPE param to use decorator or flag 
    // in cypress script files. 
+   // TODO(ruslan): Use build tags --bt with prod/znd states.
    def runLambdaTestArgs = ["yarn",
                             "lambdatest",
                             "--cy=\"--config baseUrl=${params.URL}\", retries=${params.TEST_RETRIES}",
                             "--bn",
-                            "e2e-test # (${params.URL}: ${params.REVISION_DESCRIPTION}) @${params.DEPLOYER_USERNAME}",
+                            BUILD_NAME,
                             "-p",
-                            "${params.NUM_WORKER_MACHINES}"
+                            "${params.NUM_WORKER_MACHINES}",
+                            "--sync=true", 
+                            "--bt",
+                            "e2e-test #${env.BUILD_NUMBER}",
+                            "--eof"
    ];
    
    dir('webapp/services/static') {
       withEnv(["LT_USERNAME=${lt_username}",
                "LT_ACCESS_KEY=${lt_access_key}"]) {
-         exec(runLambdaTestArgs)   
+         exec(runLambdaTestArgs); 
       }
    }
 }
 
+
 onWorker("ka-test-ec2", '6h') {
+   // TODO(ruslan): Remove @ in extraText before merging with current e2e-test pipeline, 
+   // we don't want to disturb real users.  
    notify([slack: [channel: params.SLACK_CHANNEL,
-                  when: ['FAILURE', 'UNSTABLE']]]) {
+                   sender: 'Testing Turtle',
+                   emoji: ':turtle:',
+                   extraText : 'Hey '+'@' + 
+                   params.DEPLOYER_USERNAME + 
+                   " BUILD: " + BUILD_NAME + 
+                   ') FAILED (<https://automation.lambdatest.com/logs/|Open>)',
```

Comment:
This will be a lot more readable if you use string interpolation:
```
"Hey @${params.DEPLOYER_USERNAME} build ${BUILD_NAME} FAILED (<https://automation.lambdatest.com/logs/|Open>)"
```